src/views/ScoreCalc.vue

```diff
@@ -1,275 +1,420 @@
+                <br />
+                <progress-bar :max="100" :val="playerScoreSubtotalPercentagesDX(noteType)"></progress-bar>
+              </td>
+              <td name="maimaidx-deluxe-score-result">
+                <b>{{ playerDeluxeScoreSubtotals[noteType] }}</b>
+                <br />
+                <b>({{ playerDeluxeScoreSubtotalPercentages(noteType).toFixed(4) }}%)</b>
+                <br />
+                <progress-bar :max="100" :val="playerDeluxeScoreSubtotalPercentages(noteType)"></progress-bar>
+              </td>
+            </tr>
+          </tbody>
+          <tfoot>
+            <tr>
+              <th>TOTAL</th>
+              <td>
+                <b>{{ totalNoteCount }}</b>
+              </td>
+              <td>
+                <b>{{ playerPerformanceOverview.critical }}</b>
+              </td>
+              <td>
+                <b>{{ playerPerformanceOverview.better_perfect + playerPerformanceOverview.perfect }}</b>
+              </td>
+              <td>
+                <b>{{ playerPerformanceOverview.great + playerPerformanceOverview.worse_great + playerPerformanceOverview.worse_great }}</b>
+              </td>
+              <td>
+                <b>{{ playerPerformanceOverview.good + playerPerformanceOverview.break_good }}</b>
+              </td>
+              <td>
+                <b>{{ playerPerformanceOverview.miss }}</b>
+              </td>
+              <td class="player-result">
+                <b>{{ playerScore.toFixed(0) }}</b>
+                <br />
+                <b>({{ playerAchievement.toFixed(2) }}%)</b>
+              </td>
+              <td class="player-result">
+                <b>{{ playerScoreDX.toFixed(0) }}</b>
+                <br />
+                <b>({{ playerAchievementDX.toFixed(4) }}%)</b>
+              </td>
+              <td class="player-result">
+                <b>{{ playerDeluxeScore }}</b>
+                <br />
+                <b>({{ playerDeluxeAchievement.toFixed(4) }}%)</b>
+              </td>
+            </tr>
+          </tfoot>
+        </table>
+      </div>
+    </main>
+    <footer>
+      <b>maimai Score Calculator</b> is made by
+      <a href="https://github.com/zetaraku/" target="_blank" rel="noopener">Raku Zeta</a>
+    </footer>
+  </div>
 </template>
 
-<script>
-import VueSlider from 'vue-slider-component';
-import ProgressBar from 'vue-simple-progress';
+<script type="module">
+import VueSlider from "vue-slider-component";
+import ProgressBar from "vue-simple-progress";
+
+const NOTE_TYPES = ["tap", "hold", "slide", "touch", "break"];
+const JUDGEMENT_TYPES = [
+  "critical", // Tap: 500 (1.0x), Break: 2600
+  "better_perfect", // Tap: 500 (1.0x), Break: 2550
+  "perfect", // Tap: 500 (1.0x), Break: 2500
+  "great", // Tap: 400 (0.8x), Break: 2000 (0.8x)
+  "worse_great", // Tap: 400 (0.8x), Break: 1500 (0.6x)
+  "worst_great", // Tap: 400 (0.8x), Break: 1250 (0.5x)
+  "good", // Tap: 250 (0.5x), Break: not used
```

Comment:
actually, we can use "good" to store 1250pt Great break because its score is 0.5x of its original score.
However, storing user-facing "Great" as "Good" in code can be confusing for developers.

Reply:
The base scoring inconsistency between breaks and non-break notes is a problem.
Now we need to separate breaks from the others.